pkg/validation/validation.go

```diff
@@ -169,3 +171,16 @@ func nodeGroupMachineConfigPoolSelector(nodeGroups []nropv1.NodeGroup) error {
 
 	return nil
 }
+
+// EqualNamespacedDSSlicesByName validates two slices of type NamespacedName are equal in Names
+func EqualNamespacedDSSlicesByName(s1, s2 []nropv1.NamespacedName) error {
```

Comment:
Wouldn't it be simpler to store the DaemonSets in map?

Reply:
no maps in the API, but we can use them freely internally if helps

Reply:
No longer needed, it was replaceable with reflect.DeepEqual()